### ISSUE TITLE
Add metrics service (with logging one simple metric)

### ISSUE THREAD

**Issue by melinath** [dancerfly/django-brambling#929]
Currently we only have access to metrics that heroku auto-generates. We need access to more granular data in order to tell what's happening with our application. There seem to be a few options that we could choose from:

- [Librato](https://www.librato.com/pricing)
  - This uses a non-standard format and the documentation does not look great.
- [HostedGraphite](https://elements.heroku.com/addons/hostedgraphite)
  - StatsD
  - A little pricier than datadog, and [doesn't provide a good python interface](https://devcenter.heroku.com/articles/hostedgraphite#using-with-python)
- [Datadog](https://docs.datadoghq.com/agent/basic_agent_usage/heroku/?tab=usregion)
  - Basically statsd
  - Pretty good documentation
  - Charges by the "host" which we don't really care about, but has a bit of a workaround for that for heroku. They say your "host count" will equal your dyno count.

I am most familiar with Datadog, so would be inclined to give them a shot, then switch to a different service if we have problems with them. Worst case would be that we host our own Graphite server somewhere, since the statsd format is pretty standard.

As a first step, we could start measuring total request time by view.
